admin_pages/transactions/Transactions_Admin_Page.core.php

```diff
@@ -726,14 +714,14 @@ protected function _transaction_details()
         $this->_template_args['txn_datetime']['value'] = $this->_transaction->get_i18n_datetime('TXN_timestamp');
         $this->_template_args['txn_datetime']['label'] = esc_html__('Date', 'event_espresso');
 
-        $this->_template_args['txn_status']['value'] = self::$_txn_status[ $this->_transaction->get('STS_ID') ];
+        $this->_template_args['txn_status']['value'] = self::$_txn_status[ $this->_transaction->status_ID() ];
```

Comment:
we should not be bypassing the dedicated methods on the model objects in case they get overridden

Reply:
All good reasons to change.  Unfortunately, even if the dedicated methods are refactored, any refactor would still have to account for the public `get` because its public (a mistake we made in this design).  Regardless, I think changing it where possible is still worthwhile.

Reply:
ya we unfortunately can't get rid of the public base class method and can only correct any uses we find, and you're right that an override needs to also override the parent `get()` (ex: `EE_Registration::get()`)

Maybe one day `EE_Base_Class::get()` can be deprecated during a refactor.